src/trunk.jl

```diff
@@ -84,6 +84,7 @@ function solve!(
   monotone::Bool = true,
   nm_itmax::Int = 25,
   verbose::Bool = false,
+  verbose_subsolver::Int = 0,
```

Comment:
why is one a bool and the other an int? we should have a convention.

Reply:
I used this following @amontoison 's comment above. 
Krylov.jl takes integer for the verbosity as it prints the `verbose_subsolver`-th iteration.

Reply:
yes, should we do the same here?

Reply:
Sure, it's an additional feature. I will change that.